examples/enums/enums.bal

```diff
@@ -0,0 +1,55 @@
+import ballerina/io;
+
+// An enum is defined using a module enum declaration.
+enum Color {
+  RED,
+  GREEN,
+  BLUE
+}
+
+enum Language {
+    // An enum member can explicitly specify an associated
+    // expression.
+    ENG = "English",
```

Comment:
Shall we use the standard tags used in representing these languages, "EN", "TA", and "SI".